### PR TITLE
Controlnet fix ci

### PR DESCRIPTION
**Pull request by sywangyi**
when running diffuser ci.
train_controlnet.py: error: unrecognized arguments: --sdp_on_bf16
usage: train_controlnet.py [-h] --pretrained_model_name_or_path PRETRAINED_MODEL_NAME_OR_PATH
                           [--controlnet_model_name_or_path CONTROLNET_MODEL_NAME_OR_PATH] [--revision REVISION] [--variant VARIANT]
                           [--tokenizer_name TOKENIZER_NAME] [--output_dir OUTPUT_DIR] [--cache_dir CACHE_DIR] [--seed SEED]
                           [--resolution RESOLUTION] [--train_batch_size TRAIN_BATCH_SIZE] [--num_train_epochs NUM_TRAIN_EPOCHS]
                           [--max_train_steps MAX_TRAIN_STEPS] [--checkpointing_steps CHECKPOINTING_STEPS]
                           [--checkpoints_total_limit CHECKPOINTS_TOTAL_LIMIT] [--resume_from_checkpoint RESUME_FROM_CHECKPOINT]
                           [--gradient_accumulation_steps GRADIENT_ACCUMULATION_STEPS] [--gradient_checkpointing]
                           [--learning_rate LEARNING_RATE] [--scale_lr] [--lr_scheduler LR_SCHEDULER] [--lr_warmup_steps LR_WARMUP_STEPS]
                           [--lr_num_cycles LR_NUM_CYCLES] [--lr_power LR_POWER] [--use_8bit_adam]
                           [--dataloader_num_workers DATALOADER_NUM_WORKERS] [--adam_beta1 ADAM_BETA1] [--adam_beta2 ADAM_BETA2]
                           [--adam_weight_decay ADAM_WEIGHT_DECAY] [--adam_epsilon ADAM_EPSILON] [--max_grad_norm MAX_GRAD_NORM]
                           [--push_to_hub] [--hub_token HUB_TOKEN] [--hub_model_id HUB_MODEL_ID] [--logging_dir LOGGING_DIR]
                           [--report_to REPORT_TO] [--bf16] [--dataset_name DATASET_NAME] [--dataset_config_name DATASET_CONFIG_NAME]
                           [--train_data_dir TRAIN_DATA_DIR] [--image_column IMAGE_COLUMN]
                           [--conditioning_image_column CONDITIONING_IMAGE_COLUMN] [--caption_column CAPTION_COLUMN]
                           [--max_train_samples MAX_TRAIN_SAMPLES] [--proportion_empty_prompts PROPORTION_EMPTY_PROMPTS]
                           [--validation_prompt VALIDATION_PROMPT [VALIDATION_PROMPT ...]]
                           [--validation_image VALIDATION_IMAGE [VALIDATION_IMAGE ...]] [--num_validation_images NUM_VALIDATION_IMAGES]
                           [--validation_steps VALIDATION_STEPS] [--tracker_project_name TRACKER_PROJECT_NAME]
                           [--gaudi_config_name GAUDI_CONFIG_NAME] [--throughput_warmup_steps THROUGHPUT_WARMUP_STEPS] [--use_hpu_graphs]
                           [--trust_remote_code]
train_controlnet.py: error: unrecognized arguments: --sdp_on_bf16

### CODE REVIEW

**Comment by sywangyi** [comment]
@jiminha

**Comment by regisss** [comment]
Related to #1631 
cc @skaulintel 

@sywangyi Can you try #1631 and let me know if that works?

**Comment by sywangyi** [comment]
> Related to #1631 cc @skaulintel
> 
> @sywangyi Can you try #1631 and let me know if that works?

yes. 1631 could fix the ci since it add "sdp_on_bf16" in example, but I double if --sdp_on_bf16 is working as expected since there's no sdp_on_bf16 in https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/diffusers/pipelines/controlnet/pipeline_controlnet.py#L87

**Comment by regisss** [comment]
You're right, going to comment in #1631
